src/renderer/pages/Operations/common/utils.ts

```diff
@@ -266,14 +267,20 @@ export const getSignatoryAccounts = (
   }, []);
 };
 
-export const getDestination = (tx: MultisigTransaction): Address | undefined => {
+export const getDestination = (
+  tx: MultisigTransaction,
+  chains: Record<ChainId, Chain>,
+  destinationChain?: ChainId,
+): Address | undefined => {
   if (!tx.transaction) return undefined;
 
+  const addressPrefix = (destinationChain ? chains[destinationChain] : chains[tx.transaction.chainId])?.addressPrefix;
```

Comment:
Maybe better to create variable for `(destinationChain ? chains[destinationChain] : chains[tx.transaction.chainId])` for better readability?